script.js

```diff
@@ -81,6 +83,33 @@ function checkWin() {
   return false;
 }
 
+function clearGameBoard(event) {
+  if (text.innerHTML === "PLAYER " + xGamePiece + " WINS!") {
+    text.innerHTML = "Player X turn";
+    gameSpaces[0].innerHTML = "";
```

Comment:
you chould do this in a loop
also you don't need to do this twice. Take it out of both If statements and put it at the bottom